test/unit/utils.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+require('colors')
```

Comment:
you shouldn't need to require this in a test file. chances are you need to require it in the `utils` file?